### ISSUE TITLE
Loss is NaN on first step, if data exceeds 100 sentences

### ISSUE THREAD

**Issue by MHamzaAhmad** [ide8/tacotron2_waveglow_multispeaker_gst#12]

**Comment by BlazJurisic** [comment]
same issue here

**Comment by mmehiretu2019** [comment]
I'm running into this issue as well. Did anyone figure out what the solution is?

**Comment by gashishoo7** [comment]
@BlazJurisic and others: NaN values error may be due to pytorch version. You can check the in model.py code in tacotron2 folder. In that, check decode class, def forward, if you are using torch 1.7 then just change that as below:
    
#memory, mask=~get_mask_from_lengths(memory_lengths))
memory, mask=torch.logical_not(get_mask_from_lengths(memory_lengths)).type(torch.uint8))  # Use instead for torch 1.7


Hope it helps.
